src/cgame/cg_animation.cpp

```diff
@@ -209,7 +209,7 @@ void CG_BuildAnimSkeleton( const lerpFrame_t *lf, refSkeleton_t *newSkeleton, co
 	}
 
 	// lerp between old and new animation if possible
-	if ( lf->blendlerp >= 0.0f )
+	if ( lf->blendlerp > 1.0e-6f )
```

Comment:
is there an epsilon constant? Also, for my benefit, what is the difference between lf->blendlerp > 0 and lf->blendlerp > 1.e-6f?

Lastly, if we do keep this number, please include why it was picked, what its implications are, etc.

Reply:
I changed this part to `> 0` and moved the code for truncating small values to 0 into the function that sets the `blendlerp` value.